### PR TITLE
[dev] Responsive

### PR DESCRIPTION
**Pull request by Joanartdacosta**
convert the page into responsive

### CODE REVIEW

File: app/src/components/dashboard/services/CardService.js [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import InfoCardService from "@/components/dashboard/services/InfoCardService";
 import { useState } from "react";
 import Modal from "@/components/dashboard/services/Modal";
-import Image from "next/image";
+import Image from "next/legacy/image";
```
Comment (doublethink13) [inline app/src/components/dashboard/services/CardService.js:5]: why legacy?

File: app/src/components/dashboard/services/CardService.js [hunk 2]
```diff
@@ -37,22 +37,26 @@ export default function CardService(props) {
                 alt={props.alt}
                 src={props.imgURL}
                 layout="fill"
+                unoptimized
```
Comment (doublethink13) [inline app/src/components/dashboard/services/CardService.js:40]: why unoptimized?

File: app/src/components/dashboard/services/CardService.js (hunk 1 above)
Comment (Joanartdacosta) [reply app/src/components/dashboard/services/CardService.js]: you're right, the most recent version is next/image
correction done

File: app/src/components/dashboard/services/CardService.js (hunk 2 above)
Comment (Joanartdacosta) [reply app/src/components/dashboard/services/CardService.js]: it was a typo error, deleted

**Review by doublethink13 — APPROVED** [review]

**Pull request merged**